lib/puppet/functions/stdlib/type3x.rb

```diff
@@ -0,0 +1,72 @@
+# This is an autogenerated function, ported from the original legacy version.
+# It /should work/ as is, but will not have all the benefits of the modern
+# function API. You should see the function docs to learn how to add function
+# signatures for type safety and to document this function using puppet-strings.
+#
+# https://puppet.com/docs/puppet/latest/custom_functions_ruby.html
+#
+# ---- original file header ----
+#
+# type3x.rb
+#
+# ---- original file header ----
+#
+# @summary
+#       @summary
+#      **DEPRECATED:** This function will be removed when Puppet 3 support is dropped; please migrate to the new parser's typing system.
```

Comment:
Given that V3 support was dropped in 2019, should this function be kept or dropped https://github.com/puppetlabs/puppetlabs-stdlib/commit/9e8ce942961f9d615203efa4da2de570cf9f420e#diff-12e598daecade60b04e493f45c134fec9287ac7471114d95d1995b4fb651a6c8